Client/macos/MacOSBluetoothConnector.h

```diff
@@ -21,13 +21,12 @@ class MacOSBluetoothConnector final : public IBluetoothConnector
 	virtual std::vector<BluetoothDevice> getConnectedDevices() noexcept(false);
 	char* receivedBytes;
 	int receivedLength;
-	bool wantNewData = false;
-	NSRunLoop * runLoop;
+    std::mutex receiveDataMutex;
```

Comment:
where are the imports for cond var and mutex?

Reply:
Good question, don't know why it compiles without those🤔
I've added those just to be sure :)